package-lock.json

```diff
@@ -1,6 +1,10 @@
 {
     "name": "@datawrapper/chart-core",
+<<<<<<< HEAD
+    "version": "8.3.1",
+=======
     "version": "8.4.0",
+>>>>>>> master
```

Comment:
just a heads up. That was a merge conflict at some point :)

Reply:
oopsie thought i'd resolved that. This branch has been a bit confusing, don't know how many times i've merged, must have slipped through the net at some point. Thanks!!